providers/gitlab/resources/gitlab.lr

```diff
@@ -121,11 +121,13 @@ gitlab.project.approvalSettings @defaults("approvalsBeforeMerge requirePasswordT
 
 
 // GitLab protected branch
-gitlab.project.repository.protectedBranch @defaults("name allowForcePush") {
+gitlab.project.protectedBranch @defaults("name allowForcePush") {
```

Comment:
i think that resource should also be private

Reply:
Thanks, did it for all of the new added!